### PR TITLE
Manual trigger

### LINKED ISSUE #16: manual trigger dont work as expected
- manual flag is not passed to trigger functionality resulting in incorrect status 200.
- manual trigger error handling is always trigger with manual flag resulting in errors even on successful flow
- manual trigger integration tests returns invalid results and should be fixed

### PR DESCRIPTION
**Pull request by cl-shaharshavit-rnd**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✔                                                                        |
| Chore            | ✖                                                                       |

Closes #16 

Further  information:

### CODE REVIEW

File: src/layerCreator/models/filesManager.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      }
+    }
+    return true;
+  }
+
+  private async fileExists(path: string): Promise<boolean> {
+    return fsPromise
+      .access(path, fsConstants.F_OK)
+      .then(() => true)
+      .catch((err) => {
```
Comment (asafMasa) [inline src/layerCreator/models/filesManager.ts:47]: add todo: error handling

File: src/layerCreator/models/trigger.ts [hunk 2]
```diff
@@ -91,55 +90,12 @@ export class Trigger {
           `failed to update agent-DB for for id=${metadata.source as string} version=${metadata.version as string}, error=${error.message}`
         );
       }
-    }
-    if (isManual) {
+    } else if (isManual) {
+      console.log('here!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!');
```
Comment (asafMasa) [inline src/layerCreator/models/trigger.ts:94]: console

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: src/layerCreator/models/filesManager.ts (hunk 1 above)
Comment (cl-shaharshavit-rnd) [reply src/layerCreator/models/filesManager.ts:47]: what error handling?
if there is error then the file is not available so it return false.

File: src/layerCreator/models/trigger.ts (hunk 2 above)
Comment (cl-shaharshavit-rnd) [reply src/layerCreator/models/trigger.ts]: removed

**Review by asafMasa — APPROVED** [review]

**Pull request merged**